### PR TITLE
Updated PQClean commit in `copy_from_upstream.yml`

### PR DESCRIPTION
**Pull request by xvzcf**

### CODE REVIEW

File: src/oqsconfig.h.cmake [hunk 1]
```diff
@@ -61,6 +61,13 @@
 #cmakedefine OQS_ENABLE_KEM_frodokem_976_shake 1
 #cmakedefine OQS_ENABLE_KEM_frodokem_1344_aes 1
 #cmakedefine OQS_ENABLE_KEM_frodokem_1344_shake 1
+#cmakedefine OQS_ENABLE_KEM_NTRUPRIME 1
+#cmakedefine OQS_ENABLE_KEM_ntruprime_sntrup761 1
+#cmakedefine OQS_ENABLE_KEM_ntruprime_sntrup761_avx2 1
+
```
Comment (baentsch) [inline src/oqsconfig.h.cmake:67]: Code duplication?

**Review by baentsch — CHANGES_REQUESTED** [review]
LGTM except one set of apparently duplicated define's. However, I'd suggest to either manually add back the documentation items lost as per #1358 (that's what I did after running the script previously) or to first fix #1358 permanently.

**Review by dstebila — APPROVED** [review]

File: src/oqsconfig.h.cmake (hunk 1 above)
Comment (xvzcf) [reply src/oqsconfig.h.cmake:64]: Fixed.

**Comment by xvzcf** [comment]
> LGTM except one set of apparently duplicated define's. However, I'd suggest to either manually add back the documentation items lost as per #1358 (that's what I did after running the script previously) or to first fix #1358 permanently.

I've manually added the documentation items back for now.

**Review by baentsch — APPROVED** [review]

**Pull request merged**